clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -47,7 +47,17 @@
 #include "utils.h"
 #ifdef LINUX
 // XXX: We should have the core export this to an include dir.
-#    include "../../core/unix/include/syscall.h"
+#    ifdef X86
```

Comment:
Could create a header similar to syscall.h with this logic so it can be reused; maybe call it syscall_target.h.

Reply:
I've updated the changes accordingly.